### DISCUSSION TITLE
Profile page - show/add stars

### DISCUSSION THREAD

**Discussion by theguly** [community/community#56107] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I open another member profile ( https://github.com/AnotherUser ) I can see the list of projects he has with the star-icon. The star icon is always un-checked even if the project has been starred by myself.

The tab repositories ( https://github.com/AnotherUser?tab=repositories ) has a different behaviour: it shows repo with the star icon checked when the given repo has been starred by myself.
It's also possible to star/unstar a repository clicking on the icon.

My suggestion is to apply a similar behaviour the two views, allowing users to star/unstar/see if a repo is starred from both main profile and repository tab.

**Comment by Daniel-Ric** [comment]
The repository list shown on a user’s profile Overview is largely a “static”/public view (often optimized for caching and fast loading), so it typically doesn’t include viewer-specific state like “have I starred this repo?”. Because of that, the star icon there isn’t treated as an interactive control and may appear unselected even if you’ve already starred the repo.

On the Repositories tab, GitHub loads a more interactive, personalized list, so it can reflect your own star status and lets you star/unstar directly.

So at the moment the workaround is to star/unstar from the Repositories tab or the repository page itself. That said, I agree it would be a nice UX improvement if the Overview list could optionally show the current viewer’s star state (and allow toggling), as long as performance/caching constraints allow it.
